hazelcast-spring/src/main/resources/hazelcast-spring-3.8.xsd

```diff
@@ -93,7 +93,7 @@
                                 </xs:attribute>
                             </xs:complexType>
                         </xs:element>
-                        <xs:element name="scheduled-executor-service" minOccurs="0" maxOccurs="unbounded">
+                        <xs:element name="scheduled-executor" minOccurs="0" maxOccurs="unbounded">
```

Comment:
You need to update the parser if you are changing the tag name. I named it "scheduled-executor-service" to be consistent with "durable-executor-service" but you can change it if it is better.